src/Stryker.Core/Stryker.Core.UnitTest/Mutants/CsharpMutantOrchestratorTests.cs

```diff
@@ -1948,4 +1951,315 @@ public void ShouldIncrementMutantCountUniquely()
+                string helium = "He";
+                string lithium = "Li";
+                string beryllium = "Be";
+                string boron = "B";
+                string carbon = "C";
+                string nitrogen = "N";
+                string oxygen = "O";
+                string fluorine = "F";
+                string neon = "Ne";
+                string[] elements = (StrykerNamespace.MutantControl.IsActive(11) ? [] : [
+                  hydrogen, helium, lithium, beryllium, boron, carbon, nitrogen, oxygen,
+                  fluorine, neon
+                ]);
+                foreach (var element in elements) {
+                  if (StrykerNamespace.MutantControl.IsActive(12)) {
+                  } else {
+                    if (StrykerNamespace.MutantControl.IsActive(13)) {
+                      ;
+                    } else {
+                      Console.WriteLine(element);
+                    }
+                  }
+                }
+              }
+            }
+            """;
+        ShouldMutateSourceInClassToExpected(source, expected);
+    }
+
+    [TestMethod]
+    public void ShouldMutateSpreadCollectionExpression()
+    {
+        var source = """
+                     public void M() {
+                        // Stryker disable String : Not mutation under test
+                        string[] vowels = ["a", "e", "i", "o", "u"];
+                        string[] consonants = ["b", "c", "d", "f", "g", "h", "j", "k", "l", "m",
+                                               "n", "p", "q", "r", "s", "t", "v", "w", "x", "z"];
+                        string[] alphabet = [.. vowels, .. consonants, "y"];
+                     }
+                     """;
+
+        var expected =
+            """
+            public void M() {
+              // Stryker disable String : Not mutation under test
+              if (StrykerNamespace.MutantControl.IsActive(0)) {
+              } else {
+                string[] vowels = (StrykerNamespace.MutantControl.IsActive(1) ? [] : ["a", "e", "i", "o", "u"]);
+                string[] consonants = (StrykerNamespace.MutantControl.IsActive(7) ? [] : [
+                                        "b", "c", "d", "f", "g", "h", "j", "k", "l", "m",
+                                        "n", "p", "q", "r", "s", "t", "v", "w", "x", "z"
+                                       ]);
+                string[] alphabet =
+                    (StrykerNamespace.MutantControl.IsActive(28)
+                         ? []
+                         : [..vowels, ..consonants, "y"]);
+              }
+            }
+            """;
+        ShouldMutateSourceInClassToExpected(source, expected);
+    }
+
+    [TestMethod]
+    public void ShouldMutateImplicitCollectionExpressionParameter()
+    {
+        var source = """
+                     public void M() {
+                         Iter([1]);
+                     }
+                     public IEnumerable<T> Iter<T>(IList<T> list) {
```

Comment:
I think we should strive to keep these tests as simple and clear as possible. In this case I think we don't need this method here. To keep this test simple we could remove this and only test the implicit collection part

Reply:
I mainly copied all these examples directly from the Microsoft docs on collection expressions, (https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/collection-expressions) just to have a large test area, but I agree that most of the examples are to show usage, not creation and can be trimmed down. Same with the other tests you have commented on.